static/app/components/globalModal/components.tsx

```diff
@@ -30,23 +32,52 @@ const ModalHeader = styled('header')`
   }
 `;
 
-const CloseButton = styled((p: Omit<ButtonProps, 'aria-label'>) => (
-  <Button
-    aria-label={t('Close Modal')}
-    icon={<IconClose legacySize="10px" />}
-    size="zero"
-    {...p}
-  />
-))`
+const ChonkCloseButton = chonkStyled((p: Omit<ButtonProps, 'aria-label'>) => {
+  return (
+    <Button
+      aria-label={t('Close Modal')}
+      size={'xs'}
```

Comment:
```suggestion
      size="xs"
```

there should be a lint rule for that 😅

Reply:
100%